addons/mail/static/tests/mock_server/mock_models/res_partner.js

```diff
@@ -191,7 +191,7 @@ export class ResPartner extends webModels.ResPartner {
         if (partner.id === serverState.odoobotId) {
             return "bot";
         }
-        return partner.im_status;
+        return partner.im_status || undefined;
```

Comment:
Why undefined? mock server should be parallel of python and there is no undefined in python